### PR TITLE
Miller projection doc first shot

### PR DESCRIPTION
**Pull request by julien2512**
Miller projection doc first shot

### CODE REVIEW

File: docs/source/projections/mill.rst [hunk 1]
```diff
@@ -4,7 +4,71 @@
 Miller Cylindrical
 ********************************************************************************
 
+The Miller cylindrical projection is a modified Mercator projection, proposed by Osborn Maitland Miller in 1942.
+The latitude is scaled by a factor of :math:`\frac{4}{5}`, projected according to Mercator, and then the result is multiplied by :math:`\frac{5}{4}` to retain scale along the equator.
+
++---------------------+--------------------------------------------------------------------------------+
+| **Classification**  | Neither conformal nor equal area cylindrical                                   |
++---------------------+--------------------------------------------------------------------------------+
+| **Available forms** | Forward and inverse Spherical                                                  |
```
Comment (kbevers) [inline docs/source/projections/mill.rst]: "Spherical" should be all lower case

File: docs/source/projections/mill.rst [hunk 2]
```diff
@@ -4,7 +4,71 @@
...
+
+    $ echo -100 35 | proj +proj=mill +lon_0=90w
+    -1113194.91      4061217.24
+
+Mathematical definition
+#######################
+
+The formulas describing the Miller projection are all taken from Snyder's manuals [Snyder1987]_.
+
+Spherical form
```
Comment (kbevers) [inline docs/source/projections/mill.rst]: You could leave this out, since there the spherical form is the only one for this projection

**Review by kbevers — CHANGES_REQUESTED** [review]
A few very minor details, otherwise it's good. Thanks.

File: docs/source/projections/mill.rst (hunk 1 above)
Comment (julien2512) [reply docs/source/projections/mill.rst]: Done

File: docs/source/projections/mill.rst (hunk 2 above)
Comment (julien2512) [reply docs/source/projections/mill.rst]: Done

**Comment by julien2512** [comment]
It was an easy one indeed.

**Review by kbevers — APPROVED** [review]

**Comment by julien2512** [comment]
Next will be the "Cylindrical equal-area projection".

**Pull request merged**